### DISCUSSION TITLE
Font size affects the size of the window on windows

### DISCUSSION THREAD

**Discussion by octavz** [wezterm/wezterm#570] [General]
Not sure if is a bug or by design but on Windows zooming the font will shrink/enlarge the window size, if is not full screen (alt+enter pressed) if is only maximize the window size is still changed.

I would expect the font change and not the window.

**Comment by wez** [comment]
You can configure which behavior you prefer: https://wezfurlong.org/wezterm/config/lua/config/adjust_window_size_when_changing_font_size.html

**Reply by octavz** [reply]
missed that one, thank you.
